react-components/src/hooks/use3dScenes.tsx

```diff
@@ -96,11 +58,99 @@ export const use3dScenes = (
   );
 };
 
-function createModelFromEdge(
-  newModelId: number,
-  newModelRevisionId: number,
-  properties: Cdf3dRevisionProperties
-): AddReveal3DModelOptions {
+function createMapOfScenes(
+  scenes: Array<NodeItem<Record<string, unknown>>> | Array<EdgeItem<Record<string, unknown>>>
```

Comment:
Can this be just `Array<EdgeItem<Record<string, Record<string, Cdf3dRevisionProperties>>>>`? I want to get rid of as many `as`-es as possible

Reply:
No, unfortunately not. Changed it to `NodeItem[] | EdgeItem[]` in [808e42b](https://github.com/cognitedata/reveal/pull/4131/commits/808e42b35074ad506aac0d3f47f0e32ad69050fa)